aws-amp-integration.md

```diff
@@ -9,136 +9,271 @@ Kubecost leverages the open-source Prometheus project as a time series database
 * [Amazon Managed Prometheus (AMP)](https://docs.aws.amazon.com/prometheus/latest/userguide/what-is-Amazon-Managed-Service-Prometheus.html)
 * [AMP IAM permissions and policies](https://docs.aws.amazon.com/prometheus/latest/userguide/AMP-and-IAM.html)
 
-## Install AMP
+## Architecture
+The architecture of this integration is similar to Amazon EKS cost monitoring with Kubecost, which is described in the previous blog post, with some enhancements as follows:
 
-### Prerequisites
+In this integration, an additional AWS SigV4 container is added to the cost-analyzer pod, acting as a proxy to help query metrics from Amazon Managed Service for Prometheus using the AWS SigV4 signing process. It enables passwordless authentication to reduce the risk of exposing your AWS credentials.
 
-* You have an existing AWS account.
-* You have IAM credentials to create AMP and IAM roles programmatically.
-* You have an existing Amazon EKS cluster with OIDC enabled. You can consider using the following command to enable OIDC for your existing Amazon EKS cluster:
+When the Amazon Managed Service for Prometheus integration is enabled, the bundled Prometheus server in the Kubecost Helm Chart is configured in the remote_write mode. The bundled Prometheus server sends the collected metrics to Amazon Managed Service for Prometheus using the AWS SigV4 signing process. All metrics and data are stored in Amazon Managed Service for Prometheus, and Kubecost queries the metrics directly from Amazon Managed Service for Prometheus instead of the bundled Prometheus. It helps customers not worry about maintaining and scaling the local Prometheus instance.
 
-{% hint style="info" %}
-Remember to replace `<YOUR_CLUSTER_NAME>` and `<AWS_REGION>` with your desired values.
-{% endhint %}
+There are two architectures you can deploy: 
+The Quick-Start architecture supports a small multi-cluster setup.
+The Federated architecture supports a large multi-cluster setup.
```

Comment:
Would recommend we format this as bullets or a numbered list of options

Reply:
Thank you Brett ! Fixed in the latest commit.